Sources/SmileID/Classes/Networking/Models/EnhancedKyc.swift

```diff
@@ -30,6 +30,8 @@ public struct EnhancedKycRequest: Codable {
         bankCode: String? = nil,
         callbackUrl: String?,
         partnerParams: PartnerParams,
+        sourceSdk: String,
+        sourceSdkVersion: String,
```

Comment:
I think we should set default values for these properties in the initializer parameter and remove the values where the properties are declared.

Reply:
Fixed that